### ISSUE TITLE
add more content to readme.md

### ISSUE THREAD

**Issue by mmattel** [owncloud/craft-blueprints-owncloud#2]
@dschmidt can you please add more content to readme.md so that readers and developers can better understand what this repro is about?

**Comment by dschmidt** [comment]
Yes, certainly. Everything's pretty much WIP which is why I haven't done that yet.
In short: This is a repository for the craft build tool of KDE. So far we use it on Windows only to build the client. See appveyor.yml in the client repository.
